source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -535,18 +532,26 @@ void OAuth2Filter::addResponseCookies(Http::ResponseHeaderMap& headers,
   // If opted-in, we also create a new Bearer cookie for the authorization token provided by the
   // auth server.
   if (config_->forwardBearerToken()) {
+    std::string cookie_attribute_httponly = cookie_tail_http_only;
```

Comment:
nit: the following 6 lines can be replaced with ternary operator for readablity.

Reply:
Done.